src/radical/pilot/configs/agent_summit.json

```diff
@@ -34,15 +34,16 @@
         "UpdateWorker"                : {"count" : 1},
         "AgentSchedulingComponent"    : {"count" : 1},
         "AgentStagingInputComponent"  : {"count" : 1},
-        "AgentStagingOutputComponent" : {"count" : 1}
-    },
-    
-    "agents": {
-        "agent_1": {
-            "target": "node",
-            "components": {
-                "AgentExecutingComponent" : {"count" : 1}
-            }
-        }
+        "AgentStagingOutputComponent" : {"count" : 1},
+        "AgentExecutingComponent"     : {"count" : 1}
     }
+    
+  # "agents": {
```

Comment:
This assumes our RU comment-augmented json right?

Reply:
That is correct - all json's consumed by RP are read via `ru.read_json` which makes sure of that.